packages/terra-clinical-detail-view/src/LevelContext.jsx

```diff
@@ -0,0 +1,4 @@
+import { createContext } from 'react';
+
+// eslint-disable-next-line import/prefer-default-export
+export const LevelContext = createContext(2);
```

Comment:
I think if we name this `HeadingLevelContext` here and in the filename, and elsewhere where this is used, I think it would make more sense. Thoughts?

Reply:
I agree HeadingLevelContext makes more sense. Updated here - https://github.com/cerner/terra-clinical/pull/858/commits/56ee63b0c7ee1c47753b8b681f57337c9009eb41#diff-8dc1e569e7de7601b032ceed6e9737a8b8b7a61dffd57fbae4afb27aa59a70c8R4